src/coreComponents/fileIO/Table/TableFormatter.hpp

```diff
@@ -121,13 +121,13 @@ class TableTextFormatter : public TableFormatter
 private:
 
   /// symbol for the extremity of a delemitor
-  static constexpr string_view sideCross = "+";
+  static constexpr char m_sideCross = '+';
   /// symbol to delimit a table column
-  static constexpr string_view innerCross = "+";
+  static constexpr char m_innerCross = '+';
```

Comment:
Where are `m_sideCross` and `m_innerCross` used?

Reply:
They aren't used anymore, they are just for customisation purpose
if we want to have a table with this style ``+-----+``, we can use them.

Reply:
As there is no longer code that use these contants, you must remove it. If someone will need to change the style of the tables, he will re-write them.
Keeping them would be confusing.

Reply:
OK, it's removed